source/user-manual/registering/index.rst

```diff
@@ -2,62 +2,128 @@
 
```

Comment:
I don't see the simple registration in the dropdown submenu.

Reply:
![image](https://user-images.githubusercontent.com/3830293/79741521-ae422d80-8301-11ea-9f71-45e841e737e2.png)

Reply:
After discussing the issue, the side menu structure remains the same.